frontend/apps/reader/modules/readerzooming.lua

```diff
@@ -116,9 +116,6 @@ function ReaderZooming:init()
 end
 
 function ReaderZooming:onReadSettings(config)
-    -- @TODO config file from old code base uses globalzoom_mode
```

Comment:
This todo from 2012 doesn't sound terribly relevant. Some old viewing configurations from 2011/2012 have been broken for 7 years now? I imagine people updated their settings manually by now… ;-)